compiler/noirc_evaluator/src/ssa/ir/function.rs

```diff
@@ -169,11 +169,12 @@ impl Function {
     /// want to iterate only reachable blocks.
     pub(crate) fn reachable_blocks(&self) -> BTreeSet<BasicBlockId> {
         let mut blocks = BTreeSet::new();
-        let mut stack = vec![self.entry_block];
+        let mut deque = VecDeque::new();
+        deque.push_back(self.entry_block);
```

Comment:
Can you change this back? This code does nothing since we return a BTreeSet which is ordered by block number anyway

Reply:
Was looking at older version of code - nevermind